client/patch-react.js

```diff
@@ -105,11 +106,11 @@ function wrap (fn, around) {
     return around.call(this, fn, ...args)
   }
 
-  // copy all properties
-  Object.assign(_fn, fn)
-  _fn.displayName = fn.displayName || fn.name
```

Comment:
`defineProperty` can override `name` too, we don't need to do that as far as I tested with Chrome.